docs/other/regex_status.md

```diff
@@ -0,0 +1,211 @@
+# Supported regular expression features
+
+The listings are taken from [perl regex docs](https://perldoc.perl.org/perlre). Regular expressions are applied via [metafunctions](../cpp2/metafunctions.md#regex)
+
+## Current status and planned on doing
+
+### Modifiers
+```
+ - [x] i                Do case-insensitive pattern matching. For example, "A" will match "a" under /i.
+ - [x] m                Treat the string being matched against as multiple lines. That is, change "^" and "$" from matching the start of the string's first line and the end of its last line to matching the start and end of each line within the string.
+ - [x] s                Treat the string as single line. That is, change "." to match any character whatsoever, even a newline, which normally it would not match.
+ - [x] x and xx         Extend your pattern's legibility by permitting whitespace and comments. Details in "/x and /xx"
+ - [x] n                Prevent the grouping metacharacters () from capturing. This modifier, new in 5.22, will stop $1, $2, etc... from being filled in.
```

Comment:
Probably don't need "new in 5.22," here.

Reply:
I will have another run on this and remove the perl version reference. For me this was ok, but I can understand if this throws people off.